lib/l10n_utils/__init__.py

```diff
@@ -94,28 +93,23 @@ def render(request, template, context=None, ftl_files=None, activation_files=Non
     # Every template gets its own .lang file, so figure out what it is
     # and pass it in the context
     context["template"] = template
-    context["langfile"] = get_l10n_path(template)
     context["template_source_url"] = template_source_url(template)
 
     # if `locales` is given use it as the full list of active translations
     if "active_locales" in context:
         translations = context["active_locales"]
         del context["active_locales"]
     else:
+        translations = [settings.LANGUAGE_CODE]
```

Comment:
I needed a sensible default since `translations_for_template(template)` got removed below. Let me know if this makes sense.